### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by xdemolish**

### CODE REVIEW

**Comment by helixhamin** [comment]
Actually, as I learn more, what I will do is figure out how to get the text from the consoles and output them into an expandable window, then I will completely hide them without a checkbox and people can just toggle the dsp control window to see the server status and text flow. I have seen a little on how to do it, but like everything else, I will need to do alot of research, find answers, and trial and error. I felt it was best to do my own side project before I tried tackling helping out on the DSP server itself. I know it is a different code base, but this way I can get back into understanding syntax rhythm and relearn. Not that I do not like your code cleanup, it will help me learn. I just don't want to merge until I know what is going on so I can add in the other things I want.

File: DSP Control/DSPControlForm.vb [hunk 1]
```diff
@@ -84,55 +84,17 @@
...
-                prochide.WorkingDirectory = TextBox1.Text
-                prochide.FileName = "DSConnect-server.exe"
-                Process.Start(prochide)
-                prochide.FileName = "DSGame-server.exe"
-                Process.Start(prochide)
-                prochide.FileName = "DSSearch-server.exe"
-                Process.Start(prochide)
-            End If
+        StopServer_Click(sender, e)
+        Button1_Click(sender, e)
```
Comment (helixhamin) [inline DSP Control/DSPControlForm.vb]: Okay, I like this. Much nicer. I will include in my next release. Already tested and working.

File: DSP Control/DSPControlForm.vb [hunk 2]
```diff
@@ -2,11 +2,16 @@
...
+    Dim servers(0 To 2) As String
 
     Private Sub DSPControlForm_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
         '' add all textbox names here whose value you want to persist.
         textBoxes = New TextBox() {TextBox1, TextBox2}
 
+        servers(0) = "DSConnect-server.exe"
+        servers(1) = "DSGame-server.exe"
+        servers(2) = "DSSearch-server.exe"
+
```
Comment (helixhamin) [inline DSP Control/DSPControlForm.vb]: Any idea how I can use the server array you set up and set the window properties? I like this cleanup, but I cannot figure out how to get the .WindowStyle = ProcessWindowStyle.Hidden to work.

Comment (xdemolish) [inline DSP Control/DSPControlForm.vb]: i've actually never done any vb (the 'cleanup' was just guesswork that happened to kind of work when i tested the servers starting up)

**Pull request closed without merging**